apps/src/StudioApp.js

```diff
@@ -981,6 +1000,31 @@ StudioApp.prototype.onReportComplete = function (response) {
 };
 
 /**
+ * Given a level definition, do we want to show instructions in markdown form.
+ * @param {object} level
+ * @returns {boolean}
+ */
+StudioApp.prototype.isMarkdownMode = function (level) {
+  return window.marked && level.markdownInstructions &&
+    this.LOCALE === ENGLISH_LOCALE;
+};
+
+/**
+ * @param {object} level
+ * @returns {string} Level's markdown instructions, with the markdown processed
+ *   into a string if in markdownMode
+ */
+StudioApp.prototype.getMarkdownInstructions = function (level) {
```

Comment:
I find the contract of this method alarming.  Maybe I'll learn this reading later on, but when is it useful to return unprocessed markdown instead of processed markdown?  Do we display unprocessed markdown somewhere?